CoinDock-app/src/Screens/Dashboard/Dashboard.css

```diff
@@ -0,0 +1,51 @@
+.start {
```

Comment:
I thing this classes are not being used if not used can remove this css classes

Reply:
Removed